pkg/provider-local/webhook/shoot/mutator.go

```diff
@@ -54,12 +55,26 @@ func (m *mutator) Mutate(ctx context.Context, new, _ client.Object) error {
 	}
 
 	switch x := new.(type) {
+	case *corev1.Node:
+		return m.mutateCoreDNSHpa(ctx, client)
 	case *corev1.ConfigMap:
 		switch {
 		case strings.HasPrefix(x.Name, kubeproxy.ConfigNamePrefix):
 			extensionswebhook.LogMutation(logger, x.Kind, x.Namespace, x.Name)
 			return m.mutateKubeProxyConfigMap(ctx, x)
 		}
+	case *appsv1.Deployment:
+		switch {
+		case x.Name == "coredns":
```

Comment:
```suggestion
		switch x.Name {
		case "coredns":
```
Maybe you wanted to be consistent with the other cases, but still a suggestion.

Reply:
I don't think this is required.